aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use framework::natives::GasParameters;
+
+crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framework", [
```

Comment:
Is this file generated somehow? If not, I'm slightly concerned we might make a mistake (e.g. a typo) in the future when we add a new op type.

Reply:
No this is not generated and should not be. The reason is that when we change some of the field names, we don't want the key to change or otherwise we break the on-chain gas schedule.

The macro also defines a few tests to ensure
1. All the paths (chain of field accesses) are unique
2. All keys are unique
3. All entries are covered